### PR TITLE
[CHORE]: Code Cleanup

### PR DESCRIPTION
**Pull request by justinmckenzie**
Scope:
- Gets rid of all lingering lint warnings and errors
- formatted SVG files for better readability
- Introduce `<Image />` component (stemmed from `nextjs` lint warnings)
- Small UX changes

### CODE REVIEW

File: src/pages/api/registration.ts [hunk 1]
```diff
@@ -3,10 +3,14 @@ import type { NextApiRequest, NextApiResponse } from 'next';
 import { Registration, validateRegistrationBody } from 'src/lib/registration';
 import { uploadToAirtable } from 'src/lib/airtable';
 
-export default async (req: NextApiRequest, res: NextApiResponse) => {
+export default async function register(
```
Comment (justinmckenzie) [inline src/pages/api/registration.ts:6]: @mcolella14 FYI lint complained about `anonymous export default` so I gave the function this name for now...let me know if you think there's a more suitable name for it.

Comment (mcolella14) [reply src/pages/api/registration.ts:6]: Ah ok I actually did the same thing [in this PR](https://github.com/mongodb/educators/pull/8/files#diff-3da78f93eaafcad98173138596688a4e15fbaf40e7f0e00ba7c4ecd18d221c71). So I'm fine either way!

Comment (justinmckenzie) [reply src/pages/api/registration.ts:6]: @mcolella14 oop missed that, let's go with yours and ill rebase once its merged

**Review by mcolella14 — COMMENTED** [review]
So I think I know what lint error you're referencing re: the `Image` component. I think next wants you to actually use [this built-in component](https://nextjs.org/docs/api-reference/next/image) rather than create one.

**Comment by justinmckenzie** [comment]
> So I think I know what lint error you're referencing re: the `Image` component. I think next wants you to actually use [this built-in component](https://nextjs.org/docs/api-reference/next/image) rather than create one.

Yeah, I looked into it more and followed their suggestion for image rendering using native html elements, which led me to just creating the standalone component with these elements. - https://nextjs.org/docs/messages/no-img-element. TBH Next's `<Image />` component annoys me with hardcoding the height/width or using `layout="fill"`, happy to revert to using their `<Image />` if you think that is best.

**Comment by mcolella14** [comment]
> > So I think I know what lint error you're referencing re: the `Image` component. I think next wants you to actually use [this built-in component](https://nextjs.org/docs/api-reference/next/image) rather than create one.
> 
> @mcolella14 Yeah, I looked into it more and followed their suggestion for image rendering using native html elements, which led me to just creating the standalone component with these elements. - https://nextjs.org/docs/messages/no-img-element. TBH Next's `<Image />` component annoys me with hardcoding the height/width or using `layout="fill"`, happy to revert to using their `<Image />` if you think that is best.

Ah I'm fully on board with what you did, I am also annoyed by their Image component lol.

**Review by mcolella14 — APPROVED** [review]

**Pull request merged**